ketcher-autotests/tests/Templates/Template-Library/template-library.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { Page, test } from '@playwright/test';
+import {
+  getCoordinatesOfTheMiddleOfTheScreen,
+  pressButton,
+  resetCurrentTool,
+  selectTopPanelButton,
+  STRUCTURE_LIBRARY_BUTTON_NAME,
+  takeEditorScreenshot,
+  TopPanelButton,
+} from '@utils';
+
+async function setDisplayStereoFlagsSettingToOn(page: Page) {
+  await selectTopPanelButton(TopPanelButton.Settings, page);
+  await page.getByText('Stereochemistry', { exact: true }).click();
+  await pressButton(page, 'IUPAC style');
+  await page.getByRole('option', { name: 'On' }).click();
```

Comment:
Could you suggest, why do you need exactly that option?

Reply:
Done